<ui version="4.0" >
 <class>LogManagerDlg</class>
 <widget class="QWidget" name="LogManagerDlg" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>564</width>
    <height>394</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Log Manager</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QTreeWidget" name="twLogEntries" >
     <column>
      <property name="text" >
       <string>Date</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Message</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gwSearchInLogs" >
     <property name="maximumSize" >
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title" >
      <string comment="Search for a specific log message">Search</string>
     </property>
     <property name="flat" >
      <bool>false</bool>
     </property>
     <layout class="QHBoxLayout" >
      <item>
       <layout class="QGridLayout" >
        <item row="0" column="0" colspan="2" >
         <layout class="QHBoxLayout" >
          <item>
           <widget class="QLabel" name="lbSearchLogsFor" >
            <property name="maximumSize" >
             <size>
              <width>16777215</width>
              <height>15</height>
             </size>
            </property>
            <property name="text" >
             <string comment="Search pattern">Search:</string>
            </property>
            <property name="buddy" >
             <cstring>leSearchLogs</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="KLineEdit" name="leSearchLogs" />
          </item>
          <item>
           <widget class="KPushButton" name="pbClearSearch" >
            <property name="sizePolicy" >
             <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize" >
             <size>
              <width>16</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text" >
             <string>&amp;x</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="KPushButton" name="pbLogSearch" >
            <property name="text" >
             <string comment="Start the log search">Search</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0" >
         <widget class="QCheckBox" name="cbLogError" >
          <property name="maximumSize" >
           <size>
            <width>16777215</width>
            <height>15</height>
           </size>
          </property>
          <property name="text" >
           <string>Error</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" >
         <widget class="QCheckBox" name="cbLogUpdate" >
          <property name="text" >
           <string comment="Display messages of the category 'Update'">Update</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0" >
         <widget class="QCheckBox" name="cbLogInfo" >
          <property name="maximumSize" >
           <size>
            <width>16777215</width>
            <height>15</height>
           </size>
          </property>
          <property name="text" >
           <string>Information</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1" >
         <widget class="QCheckBox" name="cbLogSettings" >
          <property name="text" >
           <string>Settings</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="2" >
         <widget class="QProgressBar" name="pbLogLoad" >
          <property name="value" >
           <number>24</number>
          </property>
         </widget>
        </item>
        <item row="4" column="0" >
         <widget class="QLabel" name="lbLogLoad" >
          <property name="text" >
           <string/>
          </property>
          <property name="wordWrap" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="4" column="1" >
         <widget class="KPushButton" name="pbAbort" >
          <property name="text" >
           <string>Cancel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="SimonGroupBox" name="gbOnlyDay" >
        <property name="title" >
         <string>Only of day</string>
        </property>
        <property name="flat" >
         <bool>false</bool>
        </property>
        <property name="checkable" >
         <bool>true</bool>
        </property>
        <property name="checked" >
         <bool>false</bool>
        </property>
        <layout class="QVBoxLayout" >
         <item>
          <widget class="QLabel" name="lbCalendar" >
           <property name="text" >
            <string>Calendar</string>
           </property>
           <property name="buddy" >
            <cstring>cwLogDay</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="SimonCalendarWidget" name="cwLogDay" >
           <property name="enabled" >
            <bool>true</bool>
           </property>
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize" >
            <size>
             <width>200</width>
             <height>0</height>
            </size>
           </property>
           <property name="firstDayOfWeek" >
            <enum>Qt::Monday</enum>
           </property>
           <property name="gridVisible" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SimonGroupBox</class>
   <extends>QGroupBox</extends>
   <header>simongroupbox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SimonCalendarWidget</class>
   <extends>QCalendarWidget</extends>
   <header>simoncalendarwidget.h</header>
  </customwidget>
 </customwidgets>
 <connections>
  <connection>
   <sender>pbClearSearch</sender>
   <signal>clicked()</signal>
   <receiver>leSearchLogs</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>149</x>
     <y>48</y>
    </hint>
    <hint type="destinationlabel" >
     <x>90</x>
     <y>47</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
